pyquil/api/_devices.py

```diff
@@ -53,11 +53,13 @@ def list_lattices(
     Query the Forest 2.0 server for its knowledge of lattices.  Optionally filters by underlying
     device name and lattice qubit count.
 
-    :return: A dictionary keyed on lattice names and valued in dictionaries of the form
-             {
-               "device_name": device_name,
-               "qubits": num_qubits
-             }
+    :return: A dictionary keyed on lattice names and valued in dictionaries of the
+        form::
```

Comment:
nitpick: is the newline before `form::` required here?

Reply:
It was the least invasive edit to make the warning go way - but without having an HTML rendering of your complete API documentation to check with I'm not 100% if this is needed or not.

Reply:
Fair enough. If it makes the validator happy, then it's fine, especially since it only pops up in a handful of places.